File: src/enemy/commands/equip.py
from .command import Command
import discord
from enemy import db

from enemy.systems import inventory


class Equip(Command):
    def process(message: discord.Message):
        equiper = db.get_user(message.author.id)
        item = db.get_item(message.author.id)

        equiper = inventory.equip(equiper, item)

        db.set_user(equiper)

File: src/commands/equip.ts
import { Message } from "discord.js";

import { unequip } from "../systems/inventory";
import { getUser, getItem, setUser } from "../db";

export function equipCommand(message: Message) {
  let equiper = getUser(message.author.id);
  const [_, info] = message.content.split(" ", 2);
  const item = getItem(parseInt(info));

  ({ equiper } = unequip(equiper, item));

  setUser(equiper);
}
